public/index.html

```diff
@@ -50,6 +50,20 @@
       name="description"
       content="Servicio Nacional de capacitacion para la industria de la construccion"
     />
+    
+    <!--SOCIAL SHARING METATAGS-->
+    
+    <meta property="og:title" content="Art Marketplace" />
+    <meta property="og:url" content="https://art-marketplace.pages.dev/" />
+    <meta property="og:description" content="Azordev's Art Marketplace" />
+    <meta property="og:image" content="https://art-marketplace.pages.dev/static/media/banner-2.dc681b211c91c47d1b75.png" />
+    <meta property="og:type" content="product" />
+    
+    <meta property="twitter:title" content="Art Marketplace" />
+    <meta property="twitter:url" content="https://art-marketplace.pages.dev/" />
+    <meta property="twitter:description" content="Azordev's Art Marketplace" />
```

Comment:
I think it can be improved, @angelik0828 do you have some idea to improve this description?

Reply:
I know, but, I didn't know what to put there

Reply:
What about the logo? @angelik0828 what do you think?

Reply:
If all is good, It can be another issue maybe?

Reply:
Sounds good to me, lets move to other issue